cmd/in-place-restart-agent/jobset-example.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: jobset.x-k8s.io/v1alpha2
```

Comment:
Shall we locate examples in `examples/` subfolder: https://github.com/kubernetes-sigs/jobset/tree/4f93865291cc69468a18b6ffb8c986b37a6a8af0/site/static/examples?